src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,11 +3,17 @@
 public class Division {
 
     public static final long division(int a, int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException("Cannot divide by zero");
+        }
```

Comment:
Checks if b is equal to 0, then if true, throw an `IllegalArgumentException` in division function